projects/go-lib/karma.conf.js

```diff
@@ -16,9 +16,9 @@ module.exports = function (config) {
     reporters: ['dots'],
     port: 9876,
     colors: true,
-    logLevel: config.LOG_INFO,
+    logLevel: config.LOG_WARN,
     autoWatch: true,
-    browsers: ['Chrome'],
+    browsers: ['ChromeHeadless'],
```

Comment:
I was using these to make testing a little easier, not really related to the PR.. but probably worth changing regardless.